src/Makefile.am

```diff
@@ -38,6 +38,8 @@ include_HEADERS += numerics/include/metaphysicl/dualexpression.h
 include_HEADERS += numerics/include/metaphysicl/dualnamedarray.h
 include_HEADERS += numerics/include/metaphysicl/dualnumber.h
 include_HEADERS += numerics/include/metaphysicl/dualnumber_decl.h
+include_HEADERS += numerics/include/metaphysicl/dualnumber_surrogate.h
```

Comment:
The naming convention here doesn't match that for MetaPhysicL, but that's okay; yours is much better.  If you want to go all the way to "dual_number_surrogate.h" that'd be fine too.  The only reason I'm not interested in adding underscores where appropriate to the existing filenames is to avoid hassle.